robotcontainer.py

```diff
@@ -1,50 +1,89 @@
-#
-# Copyright (c) FIRST and other WPILib contributors.
-# Open Source Software; you can modify and/or share it under the terms of
-# the WPILib BSD license file in the root directory of this project.
-#
+import math
+import logging
 
+logger = logging.getLogger("project212_robot")
 import wpilib
-import wpimath.controller
+from wpimath.geometry import Translation2d, Rotation2d, Pose2d
+from pathplannerlib.path import PathPlannerPath, PathConstraints, GoalEndState
 
+from swervepy import u, SwerveDrive, TrajectoryFollowerParameters
+from swervepy.impl import CoaxialSwerveModule
+
+from constants import PHYS, MECH, ELEC, OP, SW
+import subsystems.swerveComponents as swerveComponents
 import commands2
-import commands2.cmd
+#import commands2.cmd
 import commands2.button
-from commands.hangCommand import Hang, Lower, StopHang
 
+# Constants
 import constants
 from constants import OP, SW
 
-
+# Subsystem Imports
+import subsystems.shooterSubsystem
+import subsystems.intakeSubsystem
+import subsystems.armSubsystem
+import subsystems.hangSubsystem
+import subsystems.photonVisionSubsystem
+import subsystems.armSubsystem
 import subsystems.hangSubsystem
+import subsystems.swerveSubsystem
 
-from commands.hangCommand import Hang, Lower, StopHang
+# Command Imports
+from commands.shooterCommand import inwardsShooter, outwardsShooter, stopShooter
+from commands.intakeCommand import intake, outake, stopIntake, IntakeLimitCommand
+from commands.OuttakeCommand import outtakeCommand, stopBothIntakeAndShooter
+from commands.intakeCommand import intake, outake, stopIntake
+from commands.visionCommand import takeSnapShot, togglePipeline, doNothing
+import commands.armCommand
```

Comment:
Consistency is important.  If all the other command modules are written as `from commands.WhateverCommand import SomeCommand`, then `armCommand` and `hangCommand` should work the same way.